src/source.cpp

```diff
@@ -270,10 +270,11 @@ void initialize_source()
       // initialize random number seed
       int64_t id = simulation::total_gen*settings::n_particles +
         simulation::work_index[mpi::rank] + i + 1;
-      set_particle_seed(id);
+	  uint64_t prn_seeds[N_STREAMS];
```

Comment:
```suggestion
      uint64_t prn_seeds[N_STREAMS];
```

Reply:
Looks like your editor is inserting tab characters periodically. We don't use tab characters, so probably best to configure your editor to switch to spaces when working on OpenMC if possible.